DOC.md

```diff
@@ -286,7 +286,7 @@ The first parameter of `f` is the function. Its parameters are:
       parent (a `snippetNode`) will contain neither `captures` nor `env`. Those
       are only stored in the `snippet`, which can be accessed as `parent.snippet`.
 
-3-n. The `user_args` passed in `opts`.
+3. The `user_args` passed in `opts`.
```

Comment:
Oh, this isn't a typo, there might be multiple user_args, hence the `n`😅
If you want, you can mention that there may be multiple user_args, I think that might make this more clear.

Reply:
Thanks for the explaination.
I will revise this.